pkgs/os-specific/linux/trusting-trust/stage0-posix/stage0-posix-x86.nix

```diff
@@ -0,0 +1,310 @@
+# This is a translation of stage0-posix/stage0-posix/x86/mescc-tools-mini-kaem.kaem to nix
```

Comment:
We have kaem by this point anyway, so why not do it in a single kaem script rather than lots of little derivations?  (Each derivation comes with a few seconds of overhead when it finishes building, IME.)

Reply:
This one is a bit subtle. We have access to mini-kaem at this point but it doesn't support substituting environment variables. Without variables there's no way of passing in store inputs, or the `$out` path, other than as command line arguments directly. We can't dynamically generate the script in a string either as `builtins.toFile` will refuse to write anything with store paths in the content.